dcpy/models/file.py

```diff
@@ -56,12 +56,32 @@ class Geodatabase(BaseModel, extra="forbid"):
     crs: str
 
 
-# TODO: implement JSON and GEOJSON
 class Json(BaseModel):
     type: Literal["json"]
-    normalize: dict = {}
+    json_read_meta: JsonMeta
```

Comment:
@fvankrieken (and others), would love to hear your thoughts about `json_read_meta` implementation, specifically about `JsonNormalize` and `ReadJson` classes

Reply:
Since the two types have the same args and are required, I would lean with having the subfields (read_fn and read_kwargs) just be attributes of the Json class here. (with `json_read_fn`: Literal["read_json", "json_normalize"]`)